pkg/execution/queue/item.go

```diff
@@ -71,6 +71,10 @@ type Item struct {
 	// QueueName allows control over the queue name.  If not provided, this falls
 	// back to the queue mapping defined on the queue or the workflow ID of the fn.
 	QueueName *string `json:"qn,omitempty"`
+	// DisableImmediateExecution is used to store whether steps should be allowed
+	// to execute immediately upon being found, or whether they should be scheduled
+	// to ensure idempotency.
+	DisableImmediateExection bool `json:"disableImmExec,omitempty"`
```

Comment:
Can we remove this?  The queue item should list the basics (identifier & edge to run), vs "how" to run things.  That's squarely in the state store :)

Reply:
All resolved.